mytap-runtests.sql

```diff
@@ -0,0 +1,73 @@
+USE tap;
+
+DELIMITER //
+
+DROP PROCEDURE IF EXISTS _run_proc_by_prefix //
+CREATE PROCEDURE _run_proc_by_prefix(db_ TEXT, prefix_ TEXT)
+BEGIN
+    DECLARE no_more_proc_to_call BOOLEAN DEFAULT 0;
+    DECLARE proc_to_call TEXT;
+
+    DECLARE c CURSOR FOR
+        SELECT ROUTINE_NAME
+        FROM INFORMATION_SCHEMA.ROUTINES
+        WHERE
+            ROUTINE_TYPE = 'PROCEDURE'
+            AND ROUTINE_SCHEMA = db_
+            AND ROUTINE_NAME LIKE CONCAT(prefix_, '%')
+        ORDER BY ROUTINE_NAME;
+    DECLARE CONTINUE HANDLER FOR NOT FOUND SET no_more_proc_to_call = 1;
+
+    OPEN c;
+    l: LOOP
+        FETCH c INTO proc_to_call;
+        IF no_more_proc_to_call = 1 THEN
+            LEAVE l;
+        END IF;
+
+        SET @call_sql = CONCAT('CALL ', db_, '.', proc_to_call, '()');
+        PREPARE call_sql FROM @call_sql;
```

Comment:
Is it safe to use "random" `@` variables? Should we take care about potential conflicts (like rename this variable to `@mytap__call_sql`)?